### DISCUSSION TITLE
How flet work?

### DISCUSSION THREAD

**Discussion by liudonghua123** [flet-dev/flet#1006] [Q&A]
Hi, Thanks for this fantastic project, I like flutter and python, and this project brings both of them together. I read the docs and some code of `flet` but could not find how it works. 

Is the python code compiled to flutter web directly or convert to flutter code and then built for web platform? Are there any docs about the undergoing technology?

**Comment by FeodorFitsner** [comment]
Here you can find a bit information about how Flet works: https://flet.dev/blog/flet-mobile-strategy#flet-approach

Let me know if you need to elaborate anything, if you have specific questions. I'm going to work on FAQ this/next week - will include that there.

**Comment by liudonghua123** [comment]
@FeodorFitsner Thanks, I read the https://flet.dev/blog/flet-mobile-strategy#flet-approach and it seems like the roadmap or feature work of flet mobile platform. Is it the same approach for the desktop platform currently? 

And I also find an interesting project [go-flutter](https://github.com/go-flutter-desktop/go-flutter) which uses go and [glfw](https://github.com/glfw/glfw), maybe we can borrow some inspiration from it.
